Assets/MixedRealityToolkit.Tests/PlayModeTests/DemoSceneTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Input;
+using NUnit.Framework;
+
+namespace Microsoft.MixedReality.Toolkit.Tests.SanityTests
+{
+    public class DemoSceneTests
+    {
+        const string HandInteractionExamplesSceneName = "HandInteractionExamples";
+        const string HandInteractionExamplesScenePath = "MixedRealityToolkit.Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity";
+
+        const float ScenePlayDuration = 1f;
+
+        [UnityTest]
+        public IEnumerator LoadHandInteractionExamplesScene()
+        {
+            AsyncOperation loadOp = SceneManager.LoadSceneAsync(HandInteractionExamplesSceneName);
```

Comment:
It might be worth adding a comment here saying that if this has failed, the cause is probably because you have removed the HandInteractionScene from EditorBuildSettings.asset.

It's somewhat common for folks to modify this locally and sometimes even commit changes back (as we've seen due to the evolving things here)